test/Claims.t.sol

```diff
@@ -1,99 +0,0 @@
-// SPDX-License-Identifier: UNLICENSED
-pragma solidity ^0.8.20;
-
-import {Test} from "forge-std/Test.sol";
-import {Claims} from "../src/Claims.sol";
-import {IClaims} from "../src/interfaces/IClaims.sol";
-import {CurrencyLibrary, Currency} from "../src/types/Currency.sol";
-import {MockClaims} from "../src/test/MockClaims.sol";
-import {Deployers} from "./utils/Deployers.sol";
-
-contract ClaimsTest is Test, Deployers {
```

Comment:
Can we keep some of these same tests/modify them for 6909?

Reply:
We could if we're rolling our own implementation but I don't really think there's a need to have these tests if they're already in the 6909 repo?

Reply:
Using solmate's impl